specification.md

```diff
@@ -245,27 +250,27 @@ instance_uid for short). The instance_uid is recorded in each message sent from
 the Agent to the Server and from the Server to the Agent.
 
 The default URL path for the connection is /v1/opamp. The URL path MAY be
-configurable on the Agent and on the Server.
+configurable on the Client and on the Server.
 
 ## WebSocket Transport
 
 One of the supported transports for OpAMP protocol is
-[WebSocket](https://datatracker.ietf.org/doc/html/rfc6455). The Agent is a
-WebSocket client and the Server is a WebSocket Server. The Agent and the Server
+[WebSocket](https://datatracker.ietf.org/doc/html/rfc6455). The OpAMP Client
+is a WebSocket client and the Server is a WebSocket Server. The Client and the Server
 communicate using binary data WebSocket messages. The payload of each WebSocket
 message is a
 [binary serialized Protobuf](https://developers.google.com/protocol-buffers/docs/encoding)
-message. The Agent sends AgentToServer Protobuf messages and the Server sends
+message. The Client sends AgentToServer Protobuf messages and the Server sends
```

Comment:
Shouldn't we also rename AgentToServer to ClientToServer?

Reply:
No, the protocol is still very much Agent-oriented. The Server is almost exclusively concerned about the Agent capabilities, configuration and status, and hardly sees the Client as an independent entity - even when asking for Agent upgrade, it happens magically behind the scene, even though we know that it requires some sort of a supervisor.